tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -141,7 +141,7 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     granules_list = event_input["granules"]
     config = event["config"]
 
-    exclude_file_types = config.get(CONFIG_EXCLUDE_FILE_TYPES_KEY, None)
+    exclude_file_types = config.get(CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS, None)
```

Comment:
Possibly change to `CONFIG_ORCA_EXCLUDED_FILE_EXTENSIONS_KEY` to match naming convention

Reply:
updated